website/docs/cloud-docs/api-docs/team-members.mdx

```diff
@@ -44,9 +44,9 @@ The Team Membership API is used to add or remove users from teams. The [Team API
 
 -> **Note:** To add users to a team, they must first be invited to join the organization by email, and accept the invitation. This ensures that you don't accidentally add the wrong person by mistyping a username. See [the Organization Memberships API documentation](/cloud-docs/api-docs/organization-memberships) for more information.
```

Comment:
```suggestion
-> **Note:** To add users to a team, they must first receive and accept the invitation to join the organization by email. This process ensures that you do not accidentally add the wrong person by mistyping a username. Refer to [the Organization Memberships API documentation](/cloud-docs/api-docs/organization-memberships) for more information.
```

Suggest rewording the 1st sentence to avoid a passive voice style issue. Suggest adding "process" to avoid a "This" without a subject in the 2nd sentence. Change "See" to "Refer to" to avoid ableist language.